easybuild/easyblocks/s/suitesparse.py

```diff
@@ -94,8 +94,12 @@ def configure_step(self):
         try:
             for line in fileinput.input(fp, inplace=1, backup='.orig'):
                 for (k, v) in cfgvars.items():
```

Comment:
replace `k` and `v` by non-single-letter variable names (`key` and `val` will do, but more meaningful (yet still short) would be even better)